awx/main/models/inventory.py

```diff
@@ -1547,5 +1567,17 @@ class insights(PluginFileInjector):
     use_fqcn = True
 
 
+class constructed(PluginFileInjector):
+    plugin_name = 'constructed'
+    namespace = 'ansible'
+    collection = 'builtin'
+
+    def build_env(self, *args, **kwargs):
+        env = super().build_env(*args, **kwargs)
+        # Enable all types of inventory plugins so we pick up the script files from source inventories
+        del env['ANSIBLE_INVENTORY_ENABLED']
```

Comment:
Are we just removing this to enable the default enabled plugin list? Doesn't `auto` already try the list?